lua/terrortown/lang/en.lua

```diff
@@ -1622,3 +1622,200 @@ L.label_roles_credits_award_kill = "Credit reward amount for kill"
+L.header_voicechat_locational = "Locational Voicechat"
+L.header_round_setup_plyspawn = "Player Spawn Settings"
+L.header_round_setup_prep = "Round: Preparing"
+L.header_round_setup_round = "Round: Active"
+L.header_round_setup_post = "Round: Post"
+L.header_round_setup_map_duration = "Map Session"
+L.header_textchat = "Textchat"
+L.header_round_dead_players = "Dead Player Settings"
+L.header_administration_scoreboard = "Scoreboard Settings"
+L.header_hud_toggleable = "Toggleable HUD Elements"
+L.header_mapentities_prop_possession = "Prop Possession"
+L.header_mapentities_doors = "Doors"
+L.header_karma_tweaking = "Karma Tweaking"
+L.header_karma_kick = "Karma Kick and Ban"
+L.header_karma_logging = "Karma Logging"
+L.header_inventory_gernal = "Inventory Size"
+L.header_inventory_pickup = "Inventory Weapon Pickup"
+L.header_sprint_general = "Sprint Settings"
+
+L.help_killer_dna_range = "When a player is killed by another player a DNA fingerprint is left on their body. The max range convar defines the maximum distance in hammer units for DNA samples to be left. If the killer is further away, then no sample is left at the corpse."
+L.help_killer_dna_basetime = "The basetime in seconds until a DNA sample is decayed. A factor of the squared killer distance is substracted from this basetime."
+L.help_dna_radar = "The TTT2 DNA scanner shows the exact distance and direction of the selected DNA sample if equipped. However, there is also a classic DNA scanner mode that updates the selected sample with an in-world rendering everytime the cooldown has passed."
+L.help_idle = "The idle mode is used to move idle players into a forced spectator mode. To leave this mode again, they have to disable 'enforce spectator mode' in their 'gameplay' settings."
+L.help_namechange_kick = [[
+If a player changes their name during a round, this can be abused to evade being killed. Therefore it is prohibited to change the nickname during an active round.
+
+If the bantime is greater than 0, the player will be unable to reconnect to the server until that time has passed.]]
+L.help_damage_log = "Each time a player is damaged, a damage log entry is added to the console if enabled. This can also be stored to disk after a round has ended. The file is located at 'data/terrortown/logs/'"
+L.help_spawn_waves = [[
+If this variable is set to 0, all players are spawned at once. For servers with huge amounts of players, it can be beneficial to spawn the players in waves. The spawn wave interval is the time between each spawn wave. A spawn wave always spawns as many players as there are valid spawn points.
+
+Note: Make sure that the preparing time is long enough for the desired amount of spawn waves.]]
+L.help_voicechat_battery = [[
+Voicechatting with enabled voice chat battery reduces this meter. When it's empty the player can't voicechat and must wait for a few seconds for it to recharge. This can help to prevent excessive voicechat usage.
```

Comment:
Its the main usage for "müssen", must is more a should do it. In this case he really has to wait for it.
```suggestion
Voicechatting with enabled voice chat battery reduces this meter. When it's empty the player can't voicechat and has to wait for a few seconds for it to recharge. This can help to prevent excessive voicechat usage.
```

Reply:
You're right. Tht's what you get for copying this text from the TTT website :D